src/routes/communitiRoutes.js

```
@@ -1,7 +1,6 @@
 import { Router } from 'express';
-import { getPersonaList, getPersonaDetails } from '../controllers/personaController.js';
+import { getPersonaList, getCommunityPersonaDetails } from '../controllers/personaController.js';
 import { validateGetPersonas,  validateIdParam } from '../middlewares/personaValidator.js';
-import { getPersonaList, getPersonaDetails } from '../controllers/personaController.js';
 import { requireAuth } from '../middlewares/authMiddleware.js';
 import { validateGetPersonas,  validateIdParam } from '../middlewares/personaValidator.js';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'validateGetPersonas'. Consider to delete it or rename it.

'validateGetPersonas' is defined here:


(lint/suspicious/noRedeclare)